src/js/tracks/text-track-settings.js

```diff
@@ -321,17 +323,12 @@ class TextTrackSettings extends Component {
    * @method restoreSettings
    */
   restoreSettings() {
-    let err;
     let values;
 
     try {
-      [err, values] = safeParseTuple(window.localStorage.getItem('vjs-text-track-settings'));
```

Comment:
probably not necessary here but it allows us to more easily differentiate between the two errors.
Also, I wonder if safeJSONparse is used anywhere else.